src/main/kotlin/com/example/ajaxproject/model/GroupChatMessage.kt

```diff
@@ -7,25 +7,24 @@ import jakarta.persistence.GenerationType.AUTO
 import jakarta.persistence.Id
 import org.bson.types.ObjectId
 import org.springframework.data.mongodb.core.mapping.Document
-import java.time.LocalDateTime
 
-@Document("chat-message")
-data class ChatMessage(
+@Document("group-chat-message")
+data class GroupChatMessage(
     @Id
     @JsonProperty("id")
     @Column(name = "id")
     @GeneratedValue(strategy = AUTO)
     val id: ObjectId ,
 
-    @JsonProperty("chatRoom")
-    @Column(name = "chatRoom", length = 100)
-    val chatRoom: ChatRoom? = null,
+    @JsonProperty("groupChatRoom")
+    @Column(name = "groupChatRoom", length = 100)
+    val groupChatRoom: GroupChatRoom,
+
+    @JsonProperty("sender")
+    @Column(name = "sender", length = 100)
+    val sender: User,
 
     @JsonProperty("message")
     @Column(name = "message", length = 100)
     val message: String = "",
-
-    @JsonProperty("timeStamp")
-    @Column(name = "timeStamp", length = 100)
-    val timeStamp: LocalDateTime? = null)
-
+)
```

Comment:
do you use the same class for mongo model and request entity?

Reply:
No , i use different dto for request and class for mongo model